index.html

```diff
@@ -1,32 +1,164 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Setup</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    <section>
-        <header>
-            <div class="logo"><a href="#"></a>Ali-Tech</div>
-            <nav class="navbar"><img src="/img/Union.png" alt=""><a href="#"></a></nav>
-        </header>
-        <section class="main">
-            <h1 class="heading">I’m Ali Raza.<br>
-                Glad to see <br>you!</h1>
-            <p class="sub">I’m a software developer! I can help you build a product , feature or website Look through some
-                of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
-                contact me.</p>
-                <p class="contact"><a href="#">LET’S CONNECT</a></p>
-                <ul class="icon">
-                    <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt=""></a></li>
-                    <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt=""></a></li>
-                    <li class="m"><a href="#"><img src="/img/m.svg" alt=""></a></li>
-                    <li class="github"><a href="#"><img src="/img/github.svg" alt=""></a></li>
-                    <li class="v"><a href="#"><img src="/img/v.svg" alt=""></a></li>
-                </ul>
-        </section>
-    </section 
+    <header>
+        <div class="logo"><a href="#"></a>Ali-Tech</div>
+        <nav class="navbar"><img src="/img/Union.png" alt=""><a href="#"></a></nav>
+    </header>
+    <section class="main">
+        <h1 class="heading">I’m Ali Raza.<br>
+            Glad to see <br>you!</h1>
+        <p class="sub">I’m a software developer! I can help you build a product , feature or website Look through
+            some
+            of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+            contact me.</p>
+        <ul> 
+         <li><a href="#"><img src="/img/LET’S CONNECT.svg" alt="" class="contact"></a></li>
+        </ul>
+        <ul class="icon">
+            <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt=""></a></li>
+            <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt=""></a></li>
+            <li class="m"><a href="#"><img src="/img/m.svg" alt=""></a></li>
+            <li class="github"><a href="#"><img src="/img/github.svg" alt=""></a></li>
+            <li class="v"><a href="#"><img src="/img/v.svg" alt=""></a></li>
+        </ul>
+    </section>
+    <section class="workcards">
+        <div class="card card1">
+         <div class="pic1"><img src="/img/Snapshoot Portfolio (1).png" alt="Snapshoot"></div>
+              <img src="/img/Project Title (1).svg" alt="tonic" class="title">
+              <ul class="frame-container">
+                <li class="frame client">CANOPY</li>
```

Comment:
- [ ] Please make sure to use the given color for this. In Figma template, it is `#344563` because of the above mentioned reasons 👆

_Kindly apply this to all project cards_